### PR TITLE
Disable relaying feature on not supported safes

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #3004 #3003 #3018

### CODE REVIEW

File: Multisig/Logic/Models/Chain.swift [hunk 1]
```diff
@@ -415,6 +415,6 @@ extension Chain {
     }
 
     func isSupport(feature: Feature) -> Bool {
```
Comment (elgatovital) [inline Multisig/Logic/Models/Chain.swift]: isSupported would be more natural to read

Comment (biafra23) [reply Multisig/Logic/Models/Chain.swift]: `isSupported(feature: Feature) -> Bool` or `supports(feature: Feature) -> Bool`

**Review by biafra23 — APPROVED** [review]
I added one line to this PR:

In ReviewExecutionViewController
`choosePaymentVC.relaysLimit = controller.relaysLimit`

Otherwise the sheet has the wrong number.

image

**Comment by francovenica** [comment]
Testing in build Safe Stg and Prod builds, both 3.19.0 (1671) 

Looks good.
Currently only the Goerli and GNO the feature is enabled and usable
I've checked ETH, Polygon, BNB, Optimism, Arb, Avalanche and Optimism and the feature is not enabled there.

**Pull request merged**